### PR TITLE
chore: refine cordon/uncordon semantics

### PR DESCRIPTION
**Pull request by shanicky**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

as title

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

- [x] My PR contains user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: src/meta/src/manager/cluster.rs [hunk 1]
```diff
@@ -225,47 +226,28 @@ where
...
-            return Err(MetaError::invalid_parameter(
-                "Worker node does not have property",
-            ));
+        let mut txn = Transaction::default();
+        for worker in core.workers.values_mut() {
+            if worker_ids.contains(&worker.worker_node.id) {
+                if let Some(property) = worker.worker_node.property.as_mut() {
+                    let target = schedulability == Schedulability::Unschedulable;
+                    if property.is_unschedulable != target {
+                        property.is_unschedulable = target;
```
Comment (yezizp2012) [inline src/meta/src/manager/cluster.rs:242]: Should get it updated after txn success, otherwise it will cause inconsistency.

**Review by yezizp2012 — COMMENTED** [review]
Rest LGTM! Thanks for the work!

File: src/meta/src/manager/cluster.rs (hunk 1 above)
Comment (shanicky) [reply src/meta/src/manager/cluster.rs]: Thanks! I have rewritten it using VarTransaction.

**Review by yezizp2012 — APPROVED** [review]
LGTM

**Pull request merged**